todo_app/flask_config.py

```diff
@@ -6,3 +6,15 @@ class Config:
     SECRET_KEY = os.environ.get('SECRET_KEY')
     if not SECRET_KEY:
         raise ValueError("No SECRET_KEY set for Flask application. Did you follow the setup instructions?")
```

Comment:
You can actually delete any reference to SECRET_KEY now that you aren't using sessions anymore.